### PR TITLE
Add cron config for processing failed pronounced cases

### PR DESCRIPTION
**Pull request by mapingo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-1270

### Change description ###
Add nfdiv cron config for processing failed pronounced cases

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by nitinprabhu — DISMISSED** [review]

**Review by nitinprabhu — DISMISSED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  values:
+    job:
+      environment:
+        NOTIFY_TEMPLATE_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/
+        NOTIFY_TEMPLATE_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/applicant2
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/applicant2
+        S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+        FEE_API_URL: http://fees-register-api-aat.service.core-compute-aat.internal
+        IDAM_API_REDIRECT_URL: https://div-pfe-aat.service.core-compute-aat.internal/authenticated
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml:16]: none of these urls look like they should be here, they should be included in your chart and templates with `.Values.global.environment`

Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml:16]: @timja we use the same image for application and cron job and hence we are overriding it here

Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml:16]: or you using we don't need to include it here and they will be applied with what we have defined in our charts here https://github.com/hmcts/nfdiv-case-api/tree/master/charts/nfdiv-case-api ?

Comment (timja) [reply k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml:16]: yeah, it will get overridden for you

**Review by nitinprabhu — APPROVED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml (hunk 1 above)
Comment (mapingo) [reply k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml]: URLs removed
https://github.com/hmcts/cnp-flux-config/pull/12494/commits/fd7dc175c2d4bf594c0b8b2c95b3addfc284d689

File: k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  values:
+    job:
+      environment:
+        BULK_ACTION_BATCH_SIZE_MAX: 5
+        BULK_ACTION_BATCH_SIZE_MIN: 3
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: aat
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml:16]: these values are enabled by default:
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/aat-helmrelease.yaml#L10-L14

applies throughout

Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml:16]: these values are enabled by default:
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/aat-helmrelease.yaml#L10-L14

applies throughout

**Review by timja — APPROVED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml (hunk 2 above)
Comment (mapingo) [reply k8s/namespaces/nfdiv/nfdiv-cron-process-failed-pronounced-cases/aat-00.yaml]: Values have been removed
https://github.com/hmcts/cnp-flux-config/pull/12494/commits/18968d8b019f560a987fb7d9c9ee39e8a6f08421

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**